README.md

```diff
@@ -141,10 +141,18 @@ docker run --rm -e REACT_APP_HOME_SERVER_URL=https://matrix-client.matrix.org -p
 
 We also provide a [HELM chart](./charts/).
 
-> Important:
-> The collaboration features require you to configure TURN servers in your Homeserver.
-> It can work without them, but there is a high chance that it won't!
-> The official matrix.org homeserver will work; see for example how to configure them for [Synapse](https://matrix-org.github.io/synapse/latest/turn-howto.html).
+### Important notes
+
+**TURN server**
+
+The collaboration features require you to configure TURN servers in your Homeserver.
+It can work without them, but there is a high chance that it won't!
```

Comment:
I realise you didn't change this, but I don't like this wording. We should explain that while it might work to a limited extent, there might be issues.

(We should also make the widget good enough to be able to remove this or narrow it down to "it won't be instant, but it will still work except for $uncritical_features" ;) )

Reply:
Can you create another ticket for it? This PR is about rate limiting docs.